rvgo/slow/vm.go

```diff
@@ -713,38 +705,38 @@ func Step(calldata []byte, po PreimageOracle) (stateHash common.Hash, outErr err
 			default:
 				revertWithCode(0xf0012, fmt.Errorf("unrecognized resource limit lookup: %d", res))
 			}
-		case 233: // madvise - ignored
+		case fast.SysMadvise: // madvise - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 20: // epoll_create1 - ignored
+		case fast.SysEpollCreate1: // epoll_create1 - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 21: // epoll_ctl - ignored
+		case fast.SysEpollCtl: // epoll_ctl - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 59: // pipe2 - ignored
+		case fast.SysPipe2: // pipe2 - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 78: // readlinkat - ignored
+		case fast.SysReadlinnkat: // readlinkat - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 79: // newfstatat - ignored
+		case fast.SysNewfstatat: // newfstatat - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 160: // newuname - ignored
+		case fast.SysNewuname: // newuname - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 215: // munmap - ignored
+		case fast.SysMunmap: // munmap - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 278: // getrandom - ignored
+		case fast.SysGetRandom: // getrandom - ignored
 			setRegister(toU64(10), toU64(0))
 			setRegister(toU64(11), toU64(0))
-		case 261: // prlimit64 -- unsupported, we have getrlimit, is prlimit64 even called?
+		case fast.SysPrlimit64: // prlimit64 -- unsupported, we have getrlimit, is prlimit64 even called?
 			revertWithCode(0xf001ca11, fmt.Errorf("unsupported system call: %d", a7))
```

Comment:
Why don't we use `UnsupportedSyscallErr` here as well?

Reply:
just missed. thanks! I made an `UnsupportedSyscallErr` for the slow VM instead of using fast's error, because they're using different types (U64).